build.gradle

```diff
@@ -159,3 +154,15 @@ jacocoTestReport {
 
     dependsOn(test)
 }
+
+soramitsu {
+    projectGroup = 'sora'
+    docker {
+        jar = new File("build/libs/${project.name}-${project.version}.jar")
+        registry {
+            url = System.getenv('DOCKER_REGISTRY_URL')
+            username = System.getenv('DOCKER_REGISTRY_USERNAME')
+            password = System.getenv('DOCKER_REGISTRY_PASSWORD')
+        }
+    }
```

Comment:
```suggestion
    docker {
        // docker tag
        tag = System.getenv("DOCKER_TAG")
        // jar file that is used in the generated Dockerfile
        jar = new File("build/libs/${project.name}-${project.version}.jar")
    }
```
`registry` - it is the default behavior, you can remove it 
add `tag` the imige is pusged with wrong tag, should be `test-ci`

![Screenshot from 2021-03-05 19-26-19](https://user-images.githubusercontent.com/24387396/110145303-55c93100-7dea-11eb-9d6e-c8587ec34769.png)

Reply:
That was my thought at first, but seems like not. Also, that code added @rkharisov.
<img width="1534" alt="Screen Shot 2021-03-05 at 19 55 49" src="https://user-images.githubusercontent.com/30577732/110147604-f0c30a80-7dec-11eb-9c78-5764aacc9b2c.png">

Reply:
`registry` declaration can be safely removed from build script. Keeping it is not a mistake, but just unnecessary due it will be overridden with env variables.

Reply:
Ok. @BulatSaif Should I remove image with wrong tag from harbor?

Reply:
> Ok. @BulatSaif Should I remove image with wrong tag from harbor?

yes, but wait until you merge this pr, you will need to check harbor again: check that `dev` tag is present and delete `test-ci` and wrong tag.